File: Problema05/Python/main.py
# Import modules
import const
import random

# Variables
num = rand = 0

# Inputs
try:
  num = int(input("¡Adivina el número generado al azar (1-5)!: "))
except Exception as e:
  print("\nSe produjo la siguiente excepción: " + str(e))

# Process & Outputs
rand = random.randint(1,5)
print(cons.WIN + str(rand)) if num == rand else print(cons.LOSE + str(rand))

File: Problema05/Java/Main.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.Random;

class Main {
  public static void main(String[] args) {
    // Vars / Cons + Objects
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        Random r = new Random();
        int num = 0;
        int rand = 0;
        final String WIN = "¡Adivinaste! :) El número es: ";
        final String LOSE = "¡Inténtalo de nuevo! :( El número es: ";

    // Inputs
    try{
        print("¡Adivina el número generado al azar (1 a 5): ");
        num = Integer.parseInt(br.readLine());
    }catch(Exception e){
        print("\nSe generó la siguiente excepción: \n" + e.getMessage());
    }
    
    // Process & Outputs
    rand = r.nextInt(5)+1;
    print(num == rand ? WIN + rand : LOSE + rand);
  }

  public static void print(String s){
    System.out.print(s);
  }
}
